extension/neo4j_migration/src/include/main/neo4j_migration_extension.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "extension/extension.h"
+
+namespace kuzu {
+namespace neo4j_migration_extension {
+
+class Neo4jMigrationExtension final : public extension::Extension {
+public:
+    static constexpr char EXTENSION_NAME[] = "NEO4J_MIGRATION";
```

Comment:
How about `NEO4J` as the extension name? We can expand this with more functionality, e.g., scanning from neo4j in the future?

Reply:
I think neo4j-migration tool might be more concrete for now. The name `neo4j` is too generic